src/components/ui/Button/Button.tsx

```diff
@@ -2,10 +2,16 @@ import React from 'react'
 import { Button as UIButton } from '@faststore/ui'
 import type { ButtonProps } from '@faststore/ui'
 
-type Props = ButtonProps
+import './buttons.scss'
 
-function Button(props: Props) {
-  return <UIButton {...props} />
+type Props = ButtonProps & {
+  variant?: string
+}
+
+function Button({ variant = '', ...props }: Props) {
+  return (
+    <UIButton className="button" data-button-variant={variant} {...props} />
+  )
```

Comment:
If we don't pass a `variant` we still have the `data-button-variant` attribute. 
Just for avoid unnecessary markup when we don't pass the `variant:

```suggestion
function Button({ variant, ...props }: Props) {
  return (
    <UIButton
      className="button"
      data-button-variant={variant ?? undefined}
      {...props}
    />
  )
}
```